modules/browser_object_panel_ui.py

```diff
@@ -159,3 +163,27 @@ def open_bookmarks_menu(self) -> BasePage:
         with self.driver.context(self.driver.CONTEXT_CHROME):
             self.get_element("panel-ui-bookmarks").click()
         return self
+
+    def select_clear_history_option(self, option: str) -> BasePage:
+        """
+        Selects the clear history option, assumes the history panel is open.
+        """
+        with self.driver.context(self.driver.CONTEXT_CHROME):
+            self.get_element("clear-recent-history").click()
+            iframe = self.get_element("iframe")
+            BrowserActions(self.driver).switch_to_iframe_context(iframe)
+
+            with self.driver.context(self.driver.CONTEXT_CONTENT):
```

Comment:
Should we factor Dropdown out of AboutPrefs? This feels counterintuitive...

Reply:
I think so too, originally was it made for `about:prefs` dropdowns? Where would you prefer this to go? Like its own folder in the `modules`? (eg we make something called like `components` and its `modules/components` or something

Reply:
Yeah, that would work! I'm kinda sad we haven't done more of that for this project, honestly.